config/swagger.yaml

```diff
@@ -589,6 +589,64 @@ paths:
           description: Block or account not found
           schema:
             $ref: '#/definitions/Error'
+  /account/txs/{account_pubkey}:
+    get:
+      tags:
+        - internal
+      operationId: GetAccountTranactions
+      description: 'Get accounts''s transactions'
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: account_pubkey
+          description: 'Account pubkey to show transactions for'
+          required: true
+          type: string
+        - in: query
+          name: limit
+          description: 'Maximum transactions count to show'
+          required: false
+          type: integer
+          default: 20
+          maximum: 100
+          minimum: 1
+        - in: query
+          name: offset 
```

Comment:
This assumes stable sorting of internal result.
* Could you please comment how that is guaranteed - in simple case of state of node not changing;
* Could you please comment how that is guaranteed - in real world case of state of node changing (longest chain, mempool).

Reply:
Currenlty the sort is not as stable as it should be: transactions are sorted by block of inclusion but transactions in a single block are not deterministically sorted (and they should be). I'll implement a stable sort.
WRT to forks winning - I think we should return always the data currently considered as final.

Reply:
I think a general warning (for all pagination APIs) would be appropriate, there is no guarantee to get all transactions...